home3.2/src/main/java/Triangle.java

```diff
@@ -0,0 +1,36 @@
+public class Triangle {
+    private double side1;
+    private double side2;
+    private double side3;
+
+    public Triangle(double side1, double side2, double side3) {
```

Comment:
We should validate input somehow - not all triples of double values form a triangle.